src/api/spec/controllers/webui/packages/bs_requests_controller_spec.rb

```diff
@@ -1,19 +1,150 @@
 RSpec.describe Webui::Packages::BsRequestsController do
   describe 'GET #index' do
-    include_context 'a set of bs requests'
+    context 'when the user has the request_index feature flag disabled' do
```

Comment:
Do we mark spec code too with a `TODO` comment to remember to drop it after the roll-out or not? I am not sure at this point tbh.

Reply:
We'll have to revisit the spec code when we rollout this feature, as part of the specs will no longer work. So I think it's not really needed.